### ISSUE TITLE
Not able to start Trezor Suite when device connected

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#8012]
**Describe the bug**
Not able to start Trezor Suite when device connected on macARM / i was not able to reproduce on Win 10/
Experiencing this problem on 23.4.x version /23.3.x and 23.5.x OK/
Tested on multiple version and with multiple devices with model T / production and dev version/

Looks like connect is fighting over device till it crashes.
It does not happen in web version.
Can be interrupted by reconnecting device or accessing Trezor in by different app / you will get `Unrecognized device connected
` + `Authorization error: Device is used in another window` error but Trezor Suite loads.
Attaching logs.

```
2023-04-05T11:33:42.931Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/2/null
2023-04-05T11:33:43.440Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/3/null
2023-04-05T11:33:43.442Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/2/null
2023-04-05T11:33:43.946Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/3/null
2023-04-05T11:33:43.947Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/2/null
2023-04-05T11:33:44.456Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/3/null
2023-04-05T11:33:44.458Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/2/null
2023-04-05T11:33:44.967Z - DEBUG(request-interceptor): net.Socket.connect - http://127.0.0.1:21325/acquire/3/null
```

**Info:**
 - Suite version: desktop 23.4.1 (3693c7aba5ee58758c4c0acedeb812eb6c5fecd5)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.4.1 Chrome/110.0.5481.104 Electron/23.1.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: 
 - Transport: BridgeTransport 2.0.32

**Screenshots:**
<img width="1367" alt="image" src="https://user-images.githubusercontent.com/31506317/230067120-630464a1-6d3c-4a09-b318-3567e065477e.png">

https://www.youtube.com/watch?v=Cqe9NUg-FnM&feature=youtu.be
[Archive.zip](https://github.com/trezor/trezor-suite/files/11158236/Archive.zip)

**Comment by bosomt** [comment]
Looks like even after reconnect Suite sees one device as two devices 😕 

**Info:**
 - Suite version: desktop 23.4.1 (3693c7aba5ee58758c4c0acedeb812eb6c5fecd5)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.4.1 Chrome/110.0.5481.104 Electron/23.1.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.5.3 Universal (revision 2f03ace311584988d5aeab58fd1acf24ef54711a)
 - Transport: BridgeTransport 2.0.32


![image](https://user-images.githubusercontent.com/31506317/230072827-0cc4a03f-d71a-48d8-a59e-243c4c97e78b.png)

<img width="479" alt="image" src="https://user-images.githubusercontent.com/31506317/230075266-d0d6405f-f42c-4a56-9464-2446b75d2172.png">

**Comment by tomasklim** [comment]
@matejkriz maybe it is related to your weird "Bridge is not running" but device connected issue?

I've checked the WebUSB thing in Electron and it's not activated

**Comment by matejkriz** [comment]
> @matejkriz maybe it is related to your weird "Bridge is not running" but device connected issue?
> 
> I've checked the WebUSB thing in Electron and it's not activated


yes, I believe it's the same or at least related issue 
![image (4)](https://user-images.githubusercontent.com/3729633/230095641-8175487f-de05-4de0-a895-30aeb3803829.png)

**Comment by matejkriz** [comment]
also related to this https://github.com/trezor/trezor-suite/issues/4041 (when closing Suite in that state, it triggers that error)

**Comment by matejkriz** [comment]
@bosomt is it only if the device is in some "waiting" state like "Please type your passphrase on the connected host." or even in default normal state? Is it an edge case or not?

**Comment by sime** [comment]
Unable to reproduce so far with TRZ Model T on MacARM

**Comment by mroz22** [comment]
Could be related to #7870 and trezor-connect priority settings

**Comment by bosomt** [comment]
@matejkriz happens to me even when device is not in passphrase dialogue.

**Comment by matejkriz** [comment]
The issue is macOs 13.3 together with latest Suite (probably because of Node v18)

**Comment by bosomt** [comment]
~~updated OS and it looks it was fixed, can someone with 13.3.1 can confirm ?~~

**QA NOK** :(

**Info:**
 - Suite version: desktop 23.4.1 (5044d02a986e634881bbc85393c2936dacdcb6ec)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.4.1 Chrome/110.0.5481.208 Electron/23.2.1 Safari/537.36
 - OS: MacIntel **Version 13.3.1 (22E261)**
 - Screen: 1440x900
 - Device: 
 - Transport: N/A

**Comment by bosomt** [comment]
nope false alarm :(

<img width="478" alt="image" src="https://user-images.githubusercontent.com/31506317/230780058-11102785-0cab-4de9-b5e8-9bfc3f14dee6.png">

**Comment by matejkriz** [comment]
Fixed in 23.4 release only by downgrading the bridge for mac on v27, but the bug is still present in develop and we need to fix the bridge.

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 23.4.1 (6c6a0e9979bb80b2d6592214684ea3638a532df8)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/23.4.1 Chrome/110.0.5481.208 Electron/23.2.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.6.0 Universal (revision 88e1f8c7a5c7615723664c64b0a25adc0c409dee)
 - Transport: BridgeTransport 2.0.27
